src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -193,13 +190,15 @@
                          "methodName" (get-in props [:method "name"])
                          "methodVersion" (get-in props [:method "snapshotId"]))
               :headers utils/content-type=json
-              :on-done (fn [{:keys [success? get-parsed-response status-text]}]
-                         (if-not success?
-                           (swap! state assoc :error status-text :blocking-text nil)
-                           (let [template (get-parsed-response false)]
+              :on-done (fn [{:keys [success? get-parsed-response]}]
+                         (let [response (get-parsed-response false)]
```

Comment:
Moved this line up because both paths use the response.  Changed name because the error-response is not a template